department-of-veterans-affairs/caseflow
# HP Refactor copy button

Connects #4334 

### Description
This PR adds a tooltip to copy paste button on the worksheet.

### Acceptance Criteria 
- [ ]  On hover, display the following tool tip message: [Copy Veteran ID] in 14px type.
- [ ] Show the Vet ID button in its gray-dark hover state.

### Testing Plan
1. Go to Hearing Worksheet.
2. Hover over the Copy and paste button to see `Click to copy to clipboard`
3. Click Copy and paste button
